app/assets/stylesheets/application.scss

```
@@ -133,3 +133,31 @@ body {
 .landingback {
   background-color: caf0f8;
 }
+
+#general {
```

You can write all of these rules shorthand with commas like this:
```suggestion
#general, #medical, #digestion, #diary, #misc {
    height: 100vh;
}
```